.ai/skills/ai-add-instruction.md

```
@@ -0,0 +1,60 @@
+
+   ```markdown
+   ---
+   applyTo: '<applyTo value>'
+   ---
+
+   #file:../../.ai/instructions/<name>.md
+   ```
+
+4. **If Claude active** — Append to `CLAUDE.md`'s `## Conventions` section:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)